src/vscode-dts/vscode.proposed.lmTools.d.ts

```diff
@@ -120,7 +140,7 @@ declare module 'vscode' {
 
 	export interface LanguageModelTool {
 		// TODO@API should it be LanguageModelToolResult | string?
-		invoke(parameters: any, token: CancellationToken): Thenable<LanguageModelToolResult>;
+		invoke(options: LanguageModelToolInvokationOptions, token: CancellationToken): Thenable<LanguageModelToolResult>;
```

Comment:
You can bump the API version at the top of this file, and then we should adopt it in vscode-copilot at the same time.